### PR TITLE
JAMES-2771 Expose JMAP Port during benchmarks

### PR DESCRIPTION
**Pull request by blackheaven**

### CODE REVIEW

**Comment by Arsnael** [comment]
```
[4eacbad8ddc8f24c38a2f5821c309bdacb53f6fb] [ERROR] Errors: 
[4eacbad8ddc8f24c38a2f5821c309bdacb53f6fb] [ERROR]   SetMessagesMethodTest.setup:162 » ElasticsearchStatus Elasticsearch exception ...
```
test this please

File: JenkinsfileStressTests.groovy [hunk 1]
```diff
@@ -52,7 +52,7 @@ pipeline {
                             sh 'docker run -d --name=swift -p 8080:8080 -v /srv/bench-running-docker/swift:/srv/1/node/sdb1 jeantil/openstack-keystone-swift:pike'
                             sh 'docker run -d --name=rabbitmq -p 15672:15672 -p 5672:5672 rabbitmq:3.7.7-management'
 
-                            sh 'docker run -d --hostname HOSTNAME -p 25:25 -p 8000:8000 -p 110:110 -p 143:143 -p 465:465 -p 587:587 -p 993:993 --link cassandra:cassandra --link rabbitmq:rabbitmq --link elasticsearch:elasticsearch --link tika:tika --link swift:swift --name james_run -t james_run'
+                            sh 'docker run -d --hostname HOSTNAME -p 25:25-p 1080:80  -p 8000:8000 -p 110:110 -p 143:143 -p 465:465 -p 587:587 -p 993:993 --link cassandra:cassandra --link rabbitmq:rabbitmq --link elasticsearch:elasticsearch --link tika:tika --link swift:swift --name james_run -t james_run'
```
Comment (trantienduchn) [inline JenkinsfileStressTests.groovy]: I think It is missing a space s/`-p 25:25-p 1080:80`/`-p 25:25 -p 1080:80`. Could you verify it?

**Comment by blackheaven** [comment]
test this please

**Comment by rouazana** [comment]
merged

**Pull request closed without merging**